### DISCUSSION TITLE
Improve the Start Command

### DISCUSSION THREAD

**Discussion by brianvarskonst** [php-perfect/ddev-intellij-plugin#77] [Ideas]
Hello,

first of All, great Plugin!

Currently I don't saw a opportunity to extend the start command which only executes `ddev start`, I require e.g. on my local ssh keys at my daily work where I use them in the DDEV container with [`ddev auth ssh`](https://ddev.readthedocs.io/en/stable/users/cli-usage/#ssh-into-containers). So currently I start DDEV from my local terminal that way:

```shell
$ ddev start && ddev auth ssh && ddev ssh
```

Would be great to have the possibility to extend the start command, then it would be perfect for me.

**Comment by nico-loeber** [comment]
> first of All, great Plugin!

Thank you!

> I require e.g. on my local ssh keys at my daily work where I use them in the DDEV container with [ddev auth ssh](https://ddev.readthedocs.io/en/stable/users/cli-usage/#ssh-into-containers).

Same for me. I would advice you to add everything you need to run after `ddev start` to a post-start hook.

Add these lines to your `.ddev/config.yml` to execute `ddev auth ssh` automatically when DDEV ist started:

```yaml
hooks:
  post-start:
    - exec-host: "ddev auth ssh"
```

Here you can find additional information about ddev hooks: https://ddev.readthedocs.io/en/stable/users/extending-commands/

Please let me know if this solves your issue, or if you think a change to the plugin is needed still.

**Reply by brianvarskonst** [reply]
Hello @nico-loeber, 

I thought also as a possible solution, but there seems to me like a domain-driven application to the `config.yml` which should not be located at a generic configuration file for a project, but it's a solution. 👍🏻 

Thank you.
